h2/src/test/org/h2/test/testScript.sql

```diff
@@ -3196,6 +3196,18 @@ drop domain email;
 drop domain gmail;
 > ok
 
+CREATE VALUE TYPE person FOR "org.h2.samples.CustomTypes$PersonType" WITH param1,param2;
+> ok
+
+select CLASS_NAME, SQL from information_schema.custom_types;
+> CLASS_NAME                            SQL
+> ------------------------------------- ----------------------------------------------------------------------------------------
+> org.h2.samples.CustomTypes$PersonType CREATE VALUE TYPE PERSON FOR "org.h2.samples.CustomTypes$PersonType" WITH PARAM1, PARAM2
+> rows: 1
+
+drop VALUE TYPE person;
+> ok
+
 create force view address_view as select * from address;
 > ok
 
```

Comment:
Overall the binding to a column looks wrong. The following stuff have to work without any columns:
```
SELECT CAST ('1 + 3i' AS COMPLEX) + '2 + 2i'
```